fava/default-settings.conf

```diff
@@ -28,6 +28,8 @@ journal-show-type-note = True
 journal-show-type-document = True
 journal-show-type-pad = False
 journal-show-type-query = False
+journal-show-type-custom = True
+journal-show-type-budget = True
```

Comment:
I don't see a need to distinguish these two. Two toggle buttons for these seem kind of overkill